### PR TITLE
Added npm install.

### PR DESCRIPTION
**Pull request by nprail**
The NPM install was missing so I added it. It is missing on the website too.

### CODE REVIEW

**Comment by FezVrasta** [comment]
> You may install this theme using NPM, Bower or Meteor:

then

**Comment by nprail** [comment]
It says you can install it with NPM but it never says: `npm install bootstrap-material` On the website it says only: `npm install` It also says that NPM is coming soon. 

### GitHub README Screenshot:

capture

### Website Screenshot:

capture

**Comment by FezVrasta** [comment]
Sorry I just say that you should even update the phrase I've reported in my previous commit to include NPM as long as Bower and Meteor.

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Major differences:
 
 You may install this theme using Bower or Meteor:
```
Comment (FezVrasta) [inline README.md]: This row should include NPM in its list

**Comment by nprail** [comment]
Oh, I see.

**Comment by FezVrasta** [comment]
Sorry again, lol... I think the right NPM package is `bootstrap-material-design`, not `bootstrap-material`...

I thought I had already deprecated the other package, but it's seems not..

**Comment by FezVrasta** [comment]
I've unpublished the old one (`bootstrap-material`) and now the only one visible is `bootstrap-material-design`. Sorry for the confusion.

**Comment by nprail** [comment]
Okay. It was originally that but then I noticed the other one. I'll change it back.

**Comment by FezVrasta** [comment]
It looks good now, thanks a lot.

**Comment by nprail** [comment]
No problem! I love this theme! It works well.

**Pull request merged**